site/_includes/macros/cards/blog-card.njk

```diff
@@ -3,12 +3,9 @@
 {% macro blogCard(post) %}
 {% set pd = post.data %}
 <div class="blog-card rounded-lg pad-300 md:pad-400 width-full">
-  {% if pd.thumbnail %}
-  {% set thumbnail = pd.thumbnail %}
-  {% else %}
-  {% set thumbnail = pd.hero %}
-  {% endif %}
-  {% set alt = pd.alt %}
+  {% set fallbackImg = 'image/SZHNhsfjU9RbCestTGZU6N7JEWs1/Wa6Eb8lgDc7Q4ugFV2bw.svg' %}
+  {% set thumbnail = pd.thumbnail or pd.hero or fallbackImg %}
+  {% set alt = pd.alt or pd.title %}
```

Comment:
post title might have nothing to do with what is actually displayed on the image. This is not helping accessibility in any way.

Reply:
I see. Should the alt tag be a 'fallback image' for the posts that have no hero image or thumbnail?

Reply:
The alt tag is there to explain to the user, who cannot see the image, what was on the image, so that they understand the context.
Therefore, it should always describe what is actually on the picture. More info: https://web.dev/text-alternatives-for-images/

In our case, the fallback image looks like it's "Abstract blue circles".

Reply:
I got it. Thank you for clarifying. I have already updated the fallback image alt.